error-handler.js

```diff
@@ -258,9 +258,17 @@ createHandler = function createHandler(options) {
         }
       },
 
+      shouldContinue = function
```

Comment:
Should continue should not care about the state of the error handler, because the app should NEVER shut down for `503` errors.

```
shouldContinue = function shouldContinue(status) {
  var isMaintenance = status === 503;
  return isClientError(status) || isMaintenance;
}
```
}